index.html

```diff
@@ -1 +1,37 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>My Portfolio website</title>
+    <link rel="stylesheet" href="styles.css">  
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;700&display=swap"  rel="stylesheet">
+</head>
+<body>
+    <header class="nav-bar">
+        <a href="#" class="logo">My Logo</a>
+        <nav>
+            <button type="button" class="hamburger">&#9776;</button>
+        </nav>
+    </header>
+    <main>
+        <section class="content-section">
+            <div class="content">
+                <h1 class="content-text">I'm Rabecca Glad to see you!</h1>
+                <p class="content-text-p">I'm a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don't hestiate to contact me.</p>
+                <a href="#" class="social-connect">LET'S CONNECT</a>
+                <ul class="social-list">
+                    <li class="social-media-icons"><a href="https://twitter.com"><img src="twitter.png" alt=""></a></li>
+                    <li class="social-media-icons"><a href="https://www.linkedin.com"><img src="linkedin.png" alt=""></a></li>
+                    <li class="social-media-icons"><a href=""><img src="median.png" alt=""></a></li>
+                    <li class="social-media-icons"><a href="https://github.com"><img src="git.png" alt=""></a></li>
+                    <li class="social-media-icons"><a href="https://chilldigital.com.au"><img src="chill.png" alt=""></a></li>
+                </ul>
```

Comment:
- [ ] The images you are using here for social links are not available in your project. Please import all images and make sure they are visible and follow the Figma design.
![image](https://user-images.githubusercontent.com/83803414/185754345-2886a8a6-1a04-4739-9b93-ec2294d2874d.png)